theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -4,29 +4,33 @@ set -xe
 # This script sets up the PBuilder hooks to pull in our deb repositories while the package is building
 echo "--Apt Hook Setup"
 
-for pbuilderd in /etc/pbuilder/${os}*; do
-  # Cleanup any old hooks
-  sudo rm -f ${pbuilderd}/hooks/F60addforemanrepo
+# Cleanup any old hooks
+sudo rm -f /etc/pbuilder/${os}64/hooks/F60addforemanrepo
 
+if [ x$repoowner != xtheforeman ]; then
+  # Add a stagingdeb repo to the sources too, if it exists
   if wget -O/dev/null -q http://stagingdeb.theforeman.org/dists/${os}/${repoowner}-${version} ; then
-    echo "echo deb http://stagingdeb.theforeman.org/ ${os} ${repoowner}-${version} >> /etc/apt/sources.list" | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo > /dev/null
+    echo "echo deb http://stagingdeb.theforeman.org/ ${os} ${repoowner}-${version} >> /etc/apt/sources.list" | sudo tee -a /etc/pbuilder/${os}64/hooks/F60addforemanrepo > /dev/null
   fi
   if wget -O/dev/null -q http://stagingdeb.theforeman.org/dists/plugins/${repoowner} ; then
-    echo "echo deb http://stagingdeb.theforeman.org/ plugins ${repoowner} >> /etc/apt/sources.list" | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo > /dev/null
+    echo "echo deb http://stagingdeb.theforeman.org/ plugins ${repoowner} >> /etc/apt/sources.list" | sudo tee -a /etc/pbuilder/${os}64/hooks/F60addforemanrepo > /dev/null
   fi
+fi
 
-  # Use tee to get around sudo/redirection problems, with /dev/null to drop the stdout
-  echo "echo deb http://deb.theforeman.org/ ${os} ${version} >> /etc/apt/sources.list" | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo > /dev/null
-  echo "echo deb http://deb.theforeman.org/ plugins ${version} >> /etc/apt/sources.list" | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo > /dev/null
+# Use tee to get around sudo/redirection problems, with /dev/null to drop the stdout
+echo "echo deb http://deb.theforeman.org/ ${os} ${version} >> /etc/apt/sources.list" | sudo tee -a /etc/pbuilder/${os}64/hooks/F60addforemanrepo > /dev/null
+echo "echo deb http://deb.theforeman.org/ plugins ${version} >> /etc/apt/sources.list" | sudo tee -a /etc/pbuilder/${os}64/hooks/F60addforemanrepo > /dev/null
 
-  # Permit per-OS/project setups to add their own repos
-  if [ -e debian/${os}/${project}/hooks ]; then
-    sed "s/%OS%/${os}/g" debian/${os}/${project}/hooks | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo
-  fi
+# Permit per-OS/project setups to add their own repos
+if [ -e debian/${os}/${project}/hooks ]; then
+  sed "s/%OS%/${os}/g" debian/${os}/${project}/hooks | sudo tee -a ${pbuilderd}/hooks/F60addforemanrepo
+fi
+
+# Read Foreman package installation logs on failure
+echo "cat /var/log/foreman-install.log" | sudo tee /etc/pbuilder/${os}64/hooks/C10foremanlog
```

Comment:
```suggestion
echo "cat /var/log/foreman-install.log || true" | sudo tee /etc/pbuilder/${os}64/hooks/C10foremanlog
```

We shouldn't fail this if the log doesn't exist